src/commands/clear.rs

```diff
@@ -14,8 +14,25 @@ pub fn execute_clear_operation() {
 
 pub fn remove_dependencies_dir() {
     let package = PackageConfig::read();
-    if std::path::Path::new(&package.dependencies_dir).exists() {
-        for entry in WalkDir::new(package.dependencies_dir.canonicalize().unwrap()).min_depth(1) {
+    let extern_path = std::path::Path::new(&package.dependencies_dir);
+
+    let current_path = std::path::Path::new("");
+
+    // If extern is "" or ".." etc. or is a path that is an
+    // ancestor of the current directory, fail fast
+    if current_path.canonicalize().unwrap() == extern_path.canonicalize().unwrap()
+        || current_path
+            .ancestors()
+            .any(|path| path.canonicalize().unwrap() == extern_path.canonicalize().unwrap())
+    {
+        panic!(
+            "Current path {:?} would be deleted since extern path {:?} is an ancestor or empty",
+            current_path, extern_path
```

Comment:
should we .bright_yellow() these paths? that way they stand out more (and is generally what I do)